spec/tapioca/cli/gem_spec.rb

```diff
@@ -51,6 +51,7 @@ module Baz; end
       class Baz::Role
         include ::SmartProperties
         extend ::SmartProperties::ClassMethods
+        extend ::SmartProperties::ModuleMethods
```

Comment:
How come these changed when our Gemfile.lock has not changed? This seems to be a problem with how we are testing. We really should not be having smart properties' version change from underneath us.

Reply:
Ok, I fixed this with #449, so rebasing on `main` should render this unnecessary.